resources/assets/coffee/react/contest-voting/art-entry.coffee

```diff
@@ -97,3 +99,11 @@ export class ArtEntry extends React.Component
             if not isNaN(votePercentage)
               span className: "#{bn}__result-votes #{bn}__result-votes--percentage",
                 " (#{osu.formatNumber(votePercentage)}%)"
+      else if showNames
```

Comment:
no name when showing result?

Reply:
yeah, i feel like the masked entry names are less useful after the results are out and show the actual winner's name?